pkg/config/config.go

```diff
@@ -55,6 +55,16 @@ type Config struct {
 		Variables               map[string]string `toml:",omitempty"`
 		Registries              []Registry        `toml:",omitempty"`
 	} `toml:",omitempty"`
+
+	Sso struct {
```

Comment:
Rather than a single 'Sso' struct, I am thinking we could make this a map to allow us to support multiple AWS SSO configurations (for example, if a user has multiple AWS organizations).

e.g.
```go
type Config struct {
  ...
  SSO map[string]AWSSSOConfiguration
}

type AWSSSOConfiguration struct {
  StartURL string
  SSORegion string
  Prefix string
  ...
}
```

In the config file, this would then look like:
```toml

[SSO.default]
StartURL = "...."
SSORegion = "us-east-1"
...
```

In the case where a user only has *one* `AWSSSOConfiguration` we would use the current behaviour (which would solve your use case in the same way as your current implementation). In the case where they have two or more though, they could specify a particular one with `--config` (or a similar flag).